nncf/torch/compression_method_api.py

```diff
@@ -171,6 +262,34 @@ def get_transformation_layout(self, target_model: NNCFNetwork) -> PTTransformati
         self._handle_frozen_layers(target_model)
         return layout
 
+    @abstractmethod
+    def _build_controller(self, model: ModelType) -> PTCompressionAlgorithmController:
+        """
+        Simple implementation of building controller without setting builder state and loading controller's one
+        """
+
+    def build_controller(self, model: ModelType) -> PTCompressionAlgorithmController:
+        ctrl = self._build_controller(model)
+        if not isinstance(ctrl, PTCompressionAlgorithmController):
+            raise RuntimeError('Internal error: builder must create controller inherited from '
+                               '`PTCompressionAlgorithmController` class')
+        ctrl.set_builder_state_with_name(self.registered_name, self.get_state())
+        if self._ctrl_state is not None:
+            ctrl.load_state(self._ctrl_state)
+        return ctrl
```

Comment:
All PT builders should implement simple `_build_controller` method, which is called in advanced common `build_controller` one. it sets builder state to controller and loads controller state.